api/pom.xml

```diff
@@ -102,73 +125,81 @@
-                </configuration>
-            </plugin>
+           
+            <!-- Adds the manifest file created by the org.apache.felix:maven-bundle-plugin -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.2</version>
                 <configuration>
-                    <useDefaultManifestFile>true</useDefaultManifestFile>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
+                    <excludes>
+                        <exclude>**/*.java</exclude>
+                    </excludes>
                 </configuration>
             </plugin>
+            
+            <!-- 
+               Create Javadoc for API jar
+            -->
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>buildnumber-maven-plugin</artifactId>
-                <configuration>
-                    <format>{0,date,MM/dd/yyyy hh:mm aa}</format>
-                    <items>
-                        <item>timestamp</item>
-                    </items>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.4.0</version>
                 <executions>
                     <execution>
-                        <phase>validate</phase>
+                        <id>attach-api-javadocs</id>
                         <goals>
-                            <goal>create</goal>
+                            <goal>jar</goal>
                         </goals>
+                        <configuration>
+                            <source>11</source>
+                            <quiet>true</quiet>                          
+                            <additionalJOption>-Xdoclint:none</additionalJOption>
+                            <description>Jakarta Concurrency API documentation</description>
+                            <doctitle>Jakarta Concurrency API documentation</doctitle>
+                            <windowtitle>Jakarta Concurrency API documentation</windowtitle>
+                            <header><![CDATA[<br>Jakarta Concurrency API v${project.version}]]></header>
+                            <bottom><![CDATA[
+Comments to: <a href="mailto:jaspic-dev@eclipse.org">cu-dev@eclipse.org</a>.<br>
+Copyright &#169; 2020, 2022 Eclipse Foundation. All rights reserved.<br>
+Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.]]>
+                            </bottom>
+                            <docfilessubdirs>true</docfilessubdirs>
+                            <groups>
+                                <group>
+                                    <title>Jakarta Authentication API Documentation</title>
```

Comment:
Should be Jakarta Concurrency API Documentation

Reply:
Done!